### DISCUSSION TITLE
油猴安装失败《使用浏览器为EDGE》

### DISCUSSION THREAD

**Discussion by Dumplking** [the1812/Bilibili-Evolved#1497] [提问 / Q&A] [answered]
一直尝试，可以加载出安装界面，但是在安装界面会卡住，之后关闭界面，浏览器提示是否要下载该文件。就一直这样死循环。

在安装界面和正常的安装不同，那个安装的按钮没有加黑，我之后在油猴的管理面板的实用工具里面用Github里给的链接用URL进行下载，会还是会发生上述情况，但是没有下载文件的提示。但是油猴会提示我无法解析改插件。

不会发图片，好像也不能发图片。最后我下载了之后拖到了EDGE的插件库里面当一个离线的。还不知道能不能用，反正完美主义者看着感觉怪怪的。希望作者大大帮帮忙。

**Comment by the1812** [comment, marked as answer]
用主页的链接可以这么装
![image](https://user-images.githubusercontent.com/26504152/105635150-7e314780-5e9c-11eb-9f46-68acd16a6b93.png)
(图片可以直接粘贴进来的)

**Comment by Dumplking** [comment]
![image](https://user-images.githubusercontent.com/75549912/105688953-27c41780-5f35-11eb-80cb-9a1cf370b248.png)

这样吗？复制进来一堆链接。如果您可以看到就能发现我这边好像看不了图片。
希望可以描述一下安装步骤。谢谢了。

**Reply by the1812** [reply]
Tampermonkey 设置 - 实用工具 - Install From URL - 粘贴安装链接 - 安装

**Reply by Dumplking** [reply]
刚刚试了一下，还是会显示无法解析。又下载了离线预载版到Edge的自带拓展里面，同样无法打开，是不是Edge不适配？？

麻烦了，我过几天找个加速器去谷歌浏览器上下载。

**Reply by the1812** [reply]
edge要80以上, 老版本不支持

**Reply by Dumplking** [reply]
我的是最新版本的88啊啊，我在GitHub上面下载插件经常出问题，可能是网络的问题。

**Reply by Geeyun-JY3** [reply]
没网络条件用 jsDelivr 的安装源

**Reply by Dumplking** [reply]
问题解决了，是因为我开了一个拦截广告的其他插件，出现了冲突。

**Comment by lainashikaiju** [comment]
来向层主讨教经验了QAQ总是显示编译错误怎么办啊

**Reply by lainashikaiju** [reply]
使用的是360极速浏览器

**Reply by lainashikaiju** [reply]
QWQ

**Reply by Geeyun-JY3** [reply]
不支持内核太旧的浏览器：#1225

也不建议使用 360 这种套壳浏览器：#1503
https://github.com/the1812/Bilibili-Evolved#浏览器
> 不保证脚本能在["套壳类浏览器"](https://www.jianshu.com/p/67d790a8f221)或者较长时间没更新的浏览器中完美运行.

**Comment by Dumplking** [comment]
没有用过360浏览器的来着， 还是推荐使用edge，谷歌浏览器或者火狐浏览器， 其他的浏览器都会有自带的广告 我只用过EDGE的，微软自带的还是比较好用的------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;&quot;lainashikaiju&quot;<notifications@github.com&gt;
发送时间:&nbsp;2021年2月12日(星期五) 凌晨0:28
收件人:&nbsp;&quot;the1812/Bilibili-Evolved&quot;<Bilibili-Evolved@noreply.github.com&gt;;
抄送:&nbsp;&quot;这只团子不要了&quot;<860379927@qq.com&gt;;&quot;Comment&quot;<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [the1812/Bilibili-Evolved] 油猴安装失败《使用浏览器为EDGE》 (#1497)

**Comment by lainashikaiju** [comment]
可以加v聊一哈细节嘛，（悲伤）ysy15613409252------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;&quot;这只团子不要了&quot;<notifications@github.com&gt;
发送时间:&nbsp;2021年2月12日(星期五) 凌晨0:32
收件人:&nbsp;&quot;the1812/Bilibili-Evolved&quot;<Bilibili-Evolved@noreply.github.com&gt;;
抄送:&nbsp;&quot;lainashikaiju&quot;<884163172@qq.com&gt;;&quot;Comment&quot;<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [the1812/Bilibili-Evolved] 油猴安装失败《使用浏览器为EDGE》 (#1497)

**Comment by BakaJzon** [comment]
这样子：
1. 下载脚本用文本编辑器打开，复制源代码
2. Tampermonkey -> 添加新脚本 -> CTRL+A（全选） -> CTRL+V（粘贴） -> Wait a moment -> CTRL+S（保存）

应该是油猴的特性，先渲染完全部源代码再交互，然后就超时了
戳[这里](https://item.jd.com/100011978524.html)100%解决问题

**Comment by lainashikaiju** [comment]
任务解决了，万分感谢------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年2月12日(星期五) 凌晨0:32
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [the1812/Bilibili-Evolved] 油猴安装失败《使用浏览器为EDGE》 (#1497)
